cmd/utils/flags.go

```diff
@@ -393,6 +397,13 @@ func MustMakeDataDir(ctx *cli.Context) string {
 	return ""
 }
 
+func MakeKeyStoreDir(datadir string, ctx *cli.Context) string {
```

Comment:
Please add a method doc to this:

```go
// MakeKeyStoreDir resolves the folder to use for storing the account keys from the
// set command line flags, returning the explicitly requested path, or one inside
// the data directory otherwise.
func MakeKeyStoreDir(datadir string, ctx *cli.Context) string {
```